velox/type/Variant.h

```diff
@@ -29,7 +29,7 @@
 namespace facebook::velox {
 
 // Constant using in comparison of REAL and DOUBLE values.
-constexpr double kEpsilon{0.000001};
+constexpr double kEpsilon{0.00001};
```

Comment:
@majetideepak Perhaps, move this change into a separate PR. What's the motivation for it?

Reply:
@mbasmanova, I removed the explicit `round()` from the plans since we now have support for REAL and float results comparison. One value still failed between DuckDB and Velox due to precision difference. This fixed the problem. Makes sense to move it to another PR.